recipes/pcdswidgets/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pcdswidgets" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/pcdswidgets/archive/v{{ version }}.tar.gz
+  sha256: 418345fe6b801a6ca76e8e6502989769d428a945686584aee38f36efedd08344
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pydm >=1.9.0
+
+test:
+  imports:
+    - pcdswidgets
+
+about:
+  home: https://github.com/pcdshub/pcdswidgets
+  doc_url: https://pcdshub.github.io/pcdswidgets/
+  license: LicenseRef-SLAC
+  license_family: Other
```

Comment:
```suggestion
  license_family: BSD
```

Reply:
The guideline for `Other` came from this review here: https://github.com/conda-forge/staged-recipes/pull/9639/files#r327228549
Should we also change it at our other packages later?

Reply:
Yes